vignettes/v1-2020-2021-mitma-data-codebook.qmd

```diff
@@ -0,0 +1,330 @@
+---
+title: "Codebook and cookbook for v1 (2020-2021) Spanish mobility data"
+vignette: >
+  %\VignetteIndexEntry{Codebook and cookbook for v1 (2020-2021) Spanish mobility data}
+  %\VignetteEngine{quarto::html}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+execute: 
+  eval: false
+---
+
+You can view this vignette any time by running:
+
+```{r}
+spanishoddata::spod_codebook(ver = 1)
+```
+
+The mobility data v1 (2020-2021) was originally released by the [Ministry of Transport and Sustainable Mobility (MITMA)](https://www.transportes.gob.es/) [@mitma_data_v1]. The dataset is produced by [Nommon](https://www.nommon.es/){target="_blank"} using the raw data from [Orange España](https://www.orange.es/){target="_blank"}. Even though the raw data is only from one mobile phone operator, the resulting flows and and other counts of number of individuals in the data set are already resampled to be representative of the total population of Spain. The tables in the data set provide hourly flows between zones across Spain for every day of the observation period (2020-02-14 to 2021-05-09) and the number of trips per person made. This document will introduce you to the available data and provide brief code snippets on how to access it using the `spanishoddata` R package.
+
+Key sources for this codebook/cookbook include:
+
+-   [original data collection methodology in Spanish](https://cdn.mitma.gob.es/portal-web-drupal/covid-19/bigdata/mitma_-_estudio_movilidad_covid-19_informe_metodologico_v3.pdf){target="_blank"} + [automatically translated English version of methodology](https://robinlovelace.github.io/spanishoddata/assets/codebooks/mitma_-_estudio_movilidad_covid-19_informe_metodologico_v3_en.pdf){target="_blank"}
```

Comment:
the translated files are currently in pkgdown/assets/codebooks. they will be available after the pkgdown website is rebuilt